website/docs/pubsub.md

```diff
@@ -0,0 +1,222 @@
+---
+id: pubsub
+title: Pubsub
+---
+
```

Comment:
can we add that this is alpha? something like this https://github.com/open-policy-agent/gatekeeper/blob/0484f99d8857f38ef9338624c155fef5d34588da/website/docs/externaldata.md?plain=1#L8